IT.TFM/AzureDevOps/RestApi.cs

```diff
@@ -287,6 +291,67 @@ private static Uri GetRelativeUri(string url)
             return baseUri.MakeRelativeUri(uri);
         }
 
+        private static void ThrottleApi(RestResponse response)

```

Comment:
@AndreasNVI love this logic, which we should reverse engineer into our PowerShell based automation= scripts.

Reply:
Thanks, it keeps things simple and clean, and easy to read.